src/redteam/providers/crescendo/index.ts

```diff
@@ -102,6 +102,11 @@ interface CrescendoConfig {
    * Set by the layer strategy when used as: layer: { steps: [crescendo, audio] }
    */
   _perTurnLayers?: LayerConfig[];
+  /**
+   * Multi-input schema for generating multiple vars at each turn.
+   * Keys are variable names, values are descriptions.
+   */
+  inputs?: Record<string, string>;
```

Comment:
A global type would be preferable here and elsewhere `Record<string, string>` is in use